src/components/course/data/constants.js

```diff
@@ -3,6 +3,8 @@ import GetSmarterLogo from '../../../assets/icons/getsmarter-header-icon.svg';
 export const COURSE_PACING_MAP = {
   SELF_PACED: 'self_paced',
   INSTRUCTOR_PACED: 'instructor_paced',
+  INSTRUCTOR: 'instructor',
+  SELF: 'self',
```

Comment:
[clarification] Can we continue to rely on the existing `SELF_PACED` and `INSTRUCTOR_PACED`, as these are the actual values provided by the API. IMHO, it'd be confusing to have addtl properties for what is semantically the same as what already exists.